src/amo/reducers/addonsByAuthors.js

```diff
@@ -200,8 +199,7 @@ export const getAddonsForAuthorIds = (
           return addonsByAuthorsState.byAddonId[id];
         })
         .filter((addon) => {
-          const addonTypeFilter = getAddonTypeFilter(addonType);
-          return addonType ? addonTypeFilter.includes(addon.type) : true;
+          return addonType ? addon.type === addonType : true;
```

Comment:
This is hurting my brain because I don't remember why the old code was converting `addonType` and it's not obvious from the diff. Hmm.

Reply:
It looks like the old code was only filtering out non-static types, no? It wasn't using the `addonType` value at all. Maybe we can just remove this filter altogether?

Reply:
Re-reading the code and the test cases, I have the feeling that we can optionally select only add-ons of type `addonType` for a given author. This would make sense for authors who are artists and developers.

Reply:
ok, maybe add a note for QA suggesting to dig into various addons-by-author states.

Reply:
Added!